frontend/src/components/course/CourseAdminView/AssignmentsView/Grading/GradingView.tsx

```diff
@@ -115,7 +115,7 @@ const GradingView: FC<GradingViewProps> = ({ course, assignment, sectionsMap, ac
     return (
         <>
             <Stack direction="row" justifyContent="space-between" mb={1} alignItems="center" height={40}>
-                <ViewHeader view="assignments" views={["sections", "assignments", "people", "requests", "settings"]} access={access} />
+                <ViewHeaderContent view="assignments" views={["sections", "assignments", "people", "requests", "settings"]} />
```

Comment:
why is this list of view different from any of the views array we defined in `general.ts`?

Reply:
In particular, I don't follow why "surveys" is missing from the list. Isn't this the GradingView?

Reply:
Why are we using `ViewHeaderContent` instead of `ViewHeader` here?